src/pages/signin/ResendValidationForm.js

```diff
@@ -48,7 +51,13 @@ class ResendValidationForm extends React.Component {
             formSuccess: 'Link has been re-sent',
         });
 
-        resendValidationLink();
+        if (!this.props.account.validated) {
+            resendValidationLink();
+            console.debug('Account is unvalidated: Sending validation link.');
```

Comment:
Is there a reason we're printing this to the console? Seems unnecessary IMO

Reply:
Yes, because when you click "resend link" its hard to tell which flow is being called. You can check the network tab for this but its easier with two console.debug as can be seen in test steps 7 and 11. Just makes life easier when you want to debug.